docs/jetson-quickstart.md

```
@@ -0,0 +1,242 @@
+
+# 5. Health checks.
+curl http://localhost:8080/healthz   # {"status":"healthy"}
+curl http://localhost:8080/readyz    # {"status":"ready"}
+curl http://localhost:8080/v1/models # [{"name":"squeezenet","file_size":4956208,"loaded":true}]
+```
+
+## Troubleshooting
+
+### "requirements not met: cuda>=13.0..."
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)